### DISCUSSION TITLE
[BUG] no required module provides package

### DISCUSSION THREAD

**Discussion by jiangjjl** [gorilla/mux#772] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![image](https://github.com/user-attachments/assets/d567f369-3d64-4df3-af1a-b1a80cdab8db)
This is my go.mod file, but I still can't find the package when I run the Go program.
![image](https://github.com/user-attachments/assets/7eab0e31-26dd-4195-aed2-4e25cc86f85a)
This is my env

**Comment by jiangjjl** [comment]
![image](https://github.com/user-attachments/assets/a0d8d6a7-1988-4553-8613-d60ada42d929)

**Comment by jackgris** [comment]
If all those files are in root folder of your project you could run the project with this command: `go run .`
If you are in the root folder but the main function is in another folder you could run something like: `go run ./cmd/api `

[Documentation](https://go.dev/doc/tutorial/web-service-gin)
